### ISSUE TITLE
Problem during container creation deleting symlink

### ISSUE THREAD

**Issue by frankier** [indigo-dc/udocker#265]
Steps to reproduce:
```
~/.udocker/bin/udocker pull vitrivr/adampro:latest-selfcontained
~/.udocker/bin/udocker create --name=adampro vitrivr/adampro:latest-selfcontained
```

Output:
```
Error: delete not owner:  /home/frankier/.udocker/containers/e88dbc7e-b366-3b72-8a2b-551b5dcf458f/ROOT/etc/alternatives/pager.1.gz
Error: while extracting image layer
Error: creating container: e88dbc7e-b366-3b72-8a2b-551b5dcf458f
e88dbc7e-b366-3b72-8a2b-551b5dcf458f
```

The problem is that `/home/frankier/.udocker/containers/e88dbc7e-b366-3b72-8a2b-551b5dcf458f/ROOT/etc/alternatives/pager.1.gz` is a symlink that gets dereferenced when checking the uid and we end up getting the uid of `/usr/share/man/man1/more.1.gz`.

**Comment by jorge-lip** [comment]
The checking of safe paths has been improved in the devel and devel3 branches.

**Comment by jorge-lip** [comment]
Addressed in v1.3.0.
